server/src/main/java/org/opensearch/index/compositeindex/datacube/startree/StarTreeIndexSettings.java

```diff
@@ -35,7 +35,7 @@ public class StarTreeIndexSettings {
         "index.composite_index.star_tree.max_fields",
         1,
         1,
-        1,
+        2,
```

Comment:
yes, we can wait to merge this PR. Since you have taken the fixes in the #16380, logically we should not face issues in multi star tree. 
Once we would want to support the multiple star tree, we can merge this PR alongside the test.